docs/_docs/internals/syntax.md

```diff
@@ -201,14 +206,13 @@ SimpleType1       ::=  id
 Singleton         ::=  SimpleRef
                     |  SimpleLiteral
                     |  Singleton ‘.’ id
-FunArgType        ::=  [`erased`] Type
-                    |  [`erased`] ‘=>’ Type                                     PrefixOp(=>, t)
+FunArgType        ::=  Type
```

Comment:
We used to have `(A, erased B) => C` as a shorthand for 
```scala 3
PolyFunction { def apply(_: A, erased _: B): C }
```
Do we now want to require the arguments are explicitly named ?

Reply:
Yes, requiring names should be fine. It makes it easier to see the syntactic role of `erased` as a modifier.

Reply:
We can also note we used to allow `erased A => B` which would now be `(erased a: A) => B`.
But looks fine to me there too.